src/rust/lib/debug-scenes/src/visualization.rs

```diff
@@ -0,0 +1,124 @@
+#![allow(missing_docs)]
```

Comment:
I'd welcome some documentation for the scene.
How is it working? What should I expect to see?